java/memory/src/main/java/io/netty/buffer/ArrowBuf.java

```diff
@@ -493,6 +494,16 @@ public ArrowBuf retain() {
     return retain(1);
   }
 
+  @Override
+  public ByteBuf touch() {
+    return this;
```

Comment:
@zsxwing why does this return this, instead of invoking the super touch()?

Reply:
AbstractByteBuf doesn't implement it.

Reply:
The `touch` method is added for `AdvancedLeakAwareByteBuf/AdvancedLeakAwareCompositeByteBuf`. Other ByteBufs just need to delegate it or implement it as an empty method.